lib/reply.js

```diff
@@ -291,7 +291,8 @@ function sendStream (payload, res, reply) {
     sourceOpen = false
     if (err != null) {
       if (res.headersSent) {
-        res.log.error({ err }, 'response terminated with an error with headers already sent')
+        var isPrematureClose = err.code && err.code === 'ERR_STREAM_PREMATURE_CLOSE'
```

Comment:
Do we really need the first of part of that check `err.code &&` ?

Reply:
Probably not 😜!